### ISSUE TITLE
Authentication failed, incorrect username and password

### ISSUE THREAD

**Issue by vnfreezone** [Lekensteyn/apk-downloader#16]
I tried to put my account to APK DOWNLOADER's option but it say Error: Authentication failed  incorrect username and password. My account is without 2 steps verify thing. Using Chrome Version 38.0.2125.104 m

**Comment by Lekensteyn** [comment]
If you have 2FA enabled, you must use application-specific passwords, not your master password.

**Comment by vnfreezone** [comment]
No my account is without 2FA :/

**Comment by man4toman** [comment]
Same problem here.
I can login into play google but with same infos, I can't login to extension!
I get Authentication failed ....
I didn't use 2FA for my account.

**Comment by qwerty12** [comment]
Also got this message. I also got an email in my inbox from Google right away:

> We recently blocked a sign-in attempt to your Google Account
> 
> **If this was you**
> You can switch to an app made by Google such as Gmail to access your account (recommended) > or change your settings at https://www.google.com/settings/security/lesssecureapps so that your account is no longer protected by modern security standards. 
> 
> To learn more, see https://support.google.com/accounts/answer/6010255. 

Using the link given to enable "less secure apps" to access my account let me sign in through APK Downloader. Of course, I wouldn't make such a decision lightly but let's face it, we all came here to be able to use this extension.

**Comment by Lekensteyn** [comment]
I haven't tried to reproduce your issues yet.

@qwerty12 Given that this extension is open source, I can assure you that the your login data is handled safely. The password is used only once to gain an authorization token which can be used for retrieving APKs. Of course, when you use less reputable sources (such as evoz.), then you get a tracker for free and maybe even have ads injected on some day.

Trust, but verify.

**Comment by qwerty12** [comment]
> Given that this extension is open source, I can assure you that the your login data is handled safely.

My apologies, I didn't mean to imply that there's anything untrustworthy about this extension. I've seen the Evozi extension before but I didn't go through with installing it as the page looked "off" to me. When I found out today that in fact the Evozi extension is a rip-off of your work here, I was glad as I remembered seeing your name from the `bbswitch` kernel module I used to use in the past - and I've never hesitated to install that. I only wrote that last sentence as I don't really want to encourage everybody to turn off a useful Google feature - especially if it turns out that it's not needed. (Although, I did try signing in a few times with it turned on to no avail.)

**Comment by Walkman100** [comment]
I can confirm that you need to "Turn On" https://www.google.com/settings/security/lesssecureapps in order to use this. (I had previously had it on in order to use GTalk on Windows, and after switching it off I was no longer able to log in to this extension.)

@Lekensteyn Maybe put that link in your documentation?

**Comment by majinsaiyan** [comment]
I am also having the same issue on chrome version 39.0.2171.71. I don't use 2FA. Please look into it
![capture](https://cloud.githubusercontent.com/assets/8004155/5220558/1806548a-768d-11e4-93cb-b8b1f8b33604.PNG)

**Comment by majinsaiyan** [comment]
@Walkman100  Thanks mate. I thought you mentioned about disabling 2FA. Enabled that setting and now its working fine. One problem still remains. Chrome automatically disables 3rd party extensions. Any workaround for this?

**Comment by Walkman100** [comment]
You can load it if it's Unpacked (NOT the CRX) and chrome won't disable it: (On Windows)
![2014-11-28_17 48 46 506](https://cloud.githubusercontent.com/assets/5454764/5229975/89d07628-7727-11e4-9adf-a5da3c629234.png)
Unpacked first, CRX second. Running Google Chrome 34.0.1847.116 (Official Build 260972)

**Comment by mimibar** [comment]
I agree with @Walkman100. As soon as I enabled access for less secure apps I was able to authenticate in the options menu.

> I can confirm that you need to enable https://www.google.com/settings/security/lesssecureapps in order to use this. Maybe put that link in your documentation? (I had previously had it on in order to use GTalk on Windows, and after switching it off I was no longer able to log in to this extension.)

**Comment by Lekensteyn** [comment]
With 2FA enabled, I cannot enable the setting on https://www.google.com/settings/security/lesssecureapps:

> This setting is not available for accounts with 2-Step Verification enabled. Such accounts require an application-specific password for less secure apps access.

Does changing this setting still work...? Pull requests are welcome btw :-)

**Comment by Walkman100** [comment]
Maybe you have to generate an app-specific password for a less secure app...
